init/configure-excimer.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/sh
+
+# enable excimer module on init
+phpenmod -s fpm excimer
+phpenmod -s cli excimer
```

Comment:
An initialisation script that enables Excimer in the browser and on the CLI. This will run on boot when the Intranet is built.